prow/config.yaml

```diff
@@ -6256,6 +6256,19 @@ periodics:
       - "--upload=gs://kubernetes-jenkins/logs"
       - "--clean"
 
+- interval: 60m
+  agent: kubernetes
+  name: ci-kubemci-ingress-conformance
+  labels:
+    preset-service-account: true
+    preset-k8s-ssh: true
+  spec:
+    containers:
+    - args:
+      - --timeout=110
+      - --bare
+      image: gcr.io/k8s-testimages/e2e-kubemci:latest
```

Comment:
add `imagePullPolicy: Always`

wonder why presubmit did not catch this :-\

Reply:
done